### PR TITLE
Model stuff for encounters

### PR DESCRIPTION
**Pull request by danwang**

### CODE REVIEW

File: db/seed/superuser/data/encounters2.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#Encounters for patient 1 
+---
+name: France Chain
```
Comment (seshness) [inline db/seed/superuser/data/encounters2.yaml:3]: This name and the name above sound very similar.

这个名字跟那个名字一样！

**Comment by seshness** [comment]
:accept:

File: db/seed/superuser/data/encounters2.yaml (hunk 1 above)
Comment (danwang) [inline db/seed/superuser/data/encounters2.yaml:3]: 屁股

**Comment by danwang** [comment]
:heart: :heart_eyes:

**Comment by danwang** [comment]
Hehehehehe

:laughing: :+1:

**Pull request merged**